server/src/main.py

```diff
@@ -145,6 +147,17 @@ async def fastapi_lifespan(app: FastAPI):
 fastapi_app = WATcloudFastAPI(logger=logger, lifespan=fastapi_lifespan)
 transaction_lock = Lock()
 
+def resign_whitelist():
+    """
+    Function to run the cvmfs_server resign command.
+    """
+    logger.info("Running cvmfs_server resign")
+    res = subprocess.run(["cvmfs_server", "resign"], check=True)
```

Comment:
I think if check=True, then an exception is raised if returncode is nonzero.

Reply:
```
root@cvmfs-ephemeral-0:/app# cvmfs_server resign
Signing 30 day whitelist with master key... done
root@cvmfs-ephemeral-0:/app# echo $?
0
root@cvmfs-ephemeral-0:/app#
```

should be fine right?

Reply:
Oh ya I meant that the lines below this that check the return code won't get run if there's an error.